.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root
+MYSQLDB_ROOT_PASSWORD=1234
+MYSQLDB_DATABASE=book_store
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8088
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
let's add .env to .gitignore
rename this file to .env.sample
actual .env should never be in source control, it's supposed to contain secrets